### PR TITLE
use prettier defaults rather than a globally defined indent size/style

### PR DESCRIPTION
**Pull request by knoopx**
follow up of https://github.com/nushell/nana/pull/39

just `.editorconfig` was modified, everything else is prettier applying default formatting for each supported file type (html, md, json, css...).

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -4,13 +4,19 @@
 root = true
 
 [*]
-indent_style = space
```
Comment (fdncred) [inline .editorconfig:7]: I'm not sure I'm down with turning off spaces. Is there a prettier default for spaces? I'm also fine with spaces=2 on ts/tsx files.

Comment (fdncred) [reply .editorconfig:7]: i'm not a fan of using tabs at all and always prefer spaces. i'm also not a fan of just letting some system define what defaults are, which is why there was an .editorconfig to begin with.

Comment (knoopx) [reply .editorconfig:7]: here's the default config for js: https://github.com/prettier/prettier/blob/main/src/language-js/options.js
im ok with forcing space chars, none of the supported prettier languages use tabs

Comment (fdncred) [reply .editorconfig:7]: > im ok with forcing space chars, none of the supported prettier languages use tabs

ok, revert/add that back in please.

maybe the other stuff is good?

Comment (fdncred) [reply .editorconfig:7]: oh, i see you did. thanks

Comment (knoopx) [reply .editorconfig:7]: already did, check blow

**Comment by fdncred** [comment]
Thanks again @knoopx.

**Pull request merged**